bundled-libs/psr/simple-cache/composer.json

```diff
@@ -6,11 +6,11 @@
     "authors": [
         {
             "name": "PHP-FIG",
-            "homepage": "http://www.php-fig.org/"
+            "homepage": "https://www.php-fig.org/"
         }
     ],
     "require": {
-        "php": ">=5.3.0"
```

Comment:
This change is strange. I see that it was not intended, but how can we solve it? voku/simple-cache has psr/simple-cache as dependency, but sets it to 1.0 || 2.0, with only 2.0 requiring PHP 8.0. 

We should either bundle 1.0 (if compatible with PHP 8), somehow bundle both and use the one that fits, or swap out voku/simple-cache?

Reply:
I have no idea, I don't know any of them. I have only updated Smarty and included the strftime module - this was the result.
Fell free to modify the branch ;-)